libs/libc/audio/libsrc/0002-add-audio-quality-options.patch

```diff
@@ -0,0 +1,82 @@
+From 8d709a69b23efc52030824ae21db889589467d8c Mon Sep 17 00:00:00 2001
+From: Alin Jerpelea <alin.jerpelea@sony.com>
+Date: Mon, 16 Nov 2020 09:44:17 +0100
+Subject: [PATCH] add audio quality options
```

Comment:
should we upstream this path to mainline too?

Reply:
I will send the patch